etc/rc1

```diff
@@ -13,7 +13,7 @@ IFS=:
 
 for rcdir in $all_dirs; do
     for rcfile in $rcdir/rc1.d/*; do
-	[ -e $rcfile ] || continue
+	[ -f $rcfile -a -x $rcfile ] || continue
 	echo running $rcfile
         $rcfile
     done
```

Comment:
IIRC the -e check is there because we we aren't allowed to set `nullglob` in the posix shell and the glob itself is returned when it matches nothing.

But now that the files have moved to the `modprobe` subdirectory, maybe this is no longer an issue?

Reply:
Ah, that does make sense, and I did mean to remove this one after moving the modprobe files.

The change does seem like a slight improvement to me, but since it hasn't been a problem before we can just drop this commit.

Reply:
I went ahead and dropped the change since it is now orthogonal to this PR at least.